engine/backends/proverif/proverif_backend.ml

```diff
@@ -114,14 +114,74 @@ module Print = struct
       method ty_int _ = string "bitstring"
 
       method! item' item =
+        let fun_and_reduc name variants =
+          let constructor = List.hd variants in
+          match constructor with
+          | None -> empty
+          | Some constructor ->
+              let field_prefix =
+                if constructor.is_record then string ""
```

Comment:
Small nit: why not using `empty`?
```suggestion
                if constructor.is_record then empty
```

Reply:
Thanks for pointing these out! I keep forgetting what's already pre-defined in `PPrint`.